engine/src/main/java/io/camunda/zeebe/engine/Engine.java

```diff
@@ -195,6 +197,25 @@ private void handleUnexpectedError(
     }
   }
 
+  /**
+   * This method remove unuseful information from rejected records in order to not encounter the
```

Comment:
:upside_down_face: Maybe we can say "removes redundant information" instead.

```suggestion
   * This method removes redundant information from rejected records in order to avoid the
```